examples/src/bin/simple_0rtt_client.rs

```diff
@@ -9,17 +11,19 @@
 
 use std::io::{BufRead, BufReader, Write};
 use std::net::TcpStream;
+use std::str::FromStr;
 use std::sync::Arc;
+use std::{env, fs};
 
 use rustls::pki_types::ServerName;
 use rustls::RootCertStore;
 
-fn start_connection(config: &Arc<rustls::ClientConfig>, domain_name: &str) {
+fn start_connection(config: &Arc<rustls::ClientConfig>, domain_name: &str, port: u16) {
```

Comment:
The changes to `simple_0rtt_client.rs` seem OK to me, but would be best in their own independent commit.